### DISCUSSION TITLE
Feature Request: Control Panel: List Entries with Category Value as the View Name + Allowing Drag Ordering Per Category

### DISCUSSION THREAD

**Discussion by matthewstick** [craftcms/cms#8752] [Ideas]
### Description
_New Feature request_

This feature is a **deal breaker** for my clients.  This has happened a few times now, unfortunately.  As a result of Craft nor a Craft plugin offering this, I cannot use Craft as a solution for my clients and need to switch to a different platform (I would have preferred to use Craft).

My clients would like the ability to tag an entry under a multi-value category, have each category value appear in the CMS control panel as its own view, and then drag the ordering per category value.

### Steps for Feature

1. Create a Structure
2. Add a category field.  Let's say the category values of that field are A, B, C, D, and E
3. In the Entries area, have A, B, C, D, & E appear as a view.  So if you click the B view, it just shows entries that have B checked.
4. Retain sorting for entries with multi-value category selected.  So each A, B, C, D, & E areas have their own sort.  An entry can appear in multiple categories, and be sorted differently.

### Real Life Example
You make a FAQ website with 100 questions.  A single question is tagged "Troubleshooting," "Finance," and "Popular Questions".  In "Troubleshooting" you want to drag this question to 1st place.  Under "Finance," you want to drag this question to 3rd place.  And under "Popular Questions," you want to drag this question to 10th place.

**Comment by brandonkelly** [comment]
This is going to be possible in Craft 4 ;)

**Comment by matthewstick** [comment]
Glorious!
